src/pages/jia/LoginJia/LoginJia.js

```diff
@@ -1,8 +1,78 @@
-import React from "react";
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
 import "./LoginJia.scss";
 
-const LoginJia = () => {
-  return <div className="LoginJia">LoginJia</div>;
+const LoginJia = e => {
+  const [id, setId] = useState("");
+  const [pw, setPw] = useState("");
+  const [isDisabled, setIsDisabled] = useState(false);
+
+  const navigate = useNavigate();
+
+  const saveUserId = e => {
+    setId(e.target.value);
+  };
+
+  const saveUserPw = e => {
+    setPw(e.target.value);
+  };
+
+  const handleDisabled = () => {
+    alert("아이디 혹은 패스워드를 확인해주세요.");
+    setIsDisabled(true);
+  };
+  const checkAccount = e => {
+    id.includes("@") && pw.length >= 5
+      ? navigate("/main-jia")
+      : handleDisabled();
+  };
+  return (
+    <div className="LoginJia">
```

Comment:
```suggestion
    <div className="loginJia">
```
컨벤션에 맞게 잘 작성해주셨는데
대문자로 시작하는 PascalCase는 컴포넌트 이름에만 적용된다는 점 주의해주세요!